ktbridge/build.gradle.kts

```diff
@@ -37,6 +37,10 @@ kotlin {
   }
 }
 
+kotlin.sourceSets.all {
+  languageSettings.useExperimentalAnnotation("kotlin.ExperimentalStdlibApi")
```

Comment:
We need to propagate this as an annotation on our APIs rather than suppress it globally for our sources. Otherwise we hide our usage and someone upgrading their Kotlin to a newer version may see broken behavior.